lib/northstar-client.js

```diff
@@ -55,6 +55,25 @@ class NorthstarClient {
     return agent;
   }
 
+  /**
+   * Helper function to execute simple post.
+   */
+  executePost(endpoint, data, query) {
+    const agent = request
+      .post(`${this.baseURI}/${endpoint}`)
```

Comment:
Thanks for the 👀 @DFurnes -- here [was the fix, actually use `post`](https://github.com/DoSomething/northstar-js/commit/35522f2056b2bae2033b8c6bbccd9f1a22706e83#commitcomment-19083911) instead of `get`! :facepalm: